Exiled.API/Features/Player.cs

```diff
@@ -338,12 +338,12 @@ public float RunningSpeed
         /// <summary>
         /// Gets or sets walkSpeed of the player.
         /// </summary>
-        public float WalkSpeed
+        public float WalkingSpeed
```

Comment:
Changing this is a breaking change.

Reply:
Yamato made a PR on these properties about a week ago

Reply:
So I think that can be changed.